src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/TaskControllerTest.java

```diff
@@ -59,6 +59,16 @@ void should_succeed_and_return_a_204_no_content() {
         assertEquals(HttpStatus.NO_CONTENT, response.getStatusCode());
     }
 
+    @Test
+    void should_unclaim_a_task_204_no_content() {
+
+        String taskId = UUID.randomUUID().toString();
+
+        ResponseEntity<String> response = taskController.unclaimTask(taskId);
+
+        assertNotNull(response);
```

Comment:
Don't think we need to check not null.

Reply:
Also do we not want to check we called something on the camundaService or something like that?